src/graphql/listSubmissions.ts

```diff
@@ -69,6 +73,7 @@ export type Response = {
   listSubmissions: {
     total: number;
     submissions: Omit<Submission, "submitterID" | "bucketName" | "rootPath" | "history">[];
+    organizations: Organization[];
```

Comment:
Technically this is `Pick<Organization, "_id" | "name">`, but this isn't a big deal ATM.

Reply:
Agreed, updated in [99a458b](https://github.com/CBIIT/crdc-datahub-ui/pull/516/commits/99a458b2d8e1a512ce26fe037fc351052e1e76e4).